src/payment/strategies/offsite/offsite-payment-strategy.ts

```diff
@@ -32,8 +33,12 @@ export default class OffsitePaymentStrategy implements PaymentStrategy {
                     payment.methodId,
                     payment.gatewayId,
                     instrumentId,
-                    shouldSaveInstrument))
-            );
+                    shouldSaveInstrument,
+                    {
+                        ...getBrowserInfo(),
```

Comment:
There's no need for the execute method to know about `getBrowserInfo()`. It is always good to try and hide as much as possible the implementation details. Just send it for all `offsitePayments`.